README.md

```diff
@@ -221,7 +221,7 @@ Dynamic settings can include the following parameters:
-| models.<model_name>.upstreams                 | `endpoint`: Model endpoint.<br />`key`: Your API key.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                     |
+| models.<model_name>.upstreams                 | `endpoint`: Model endpoint.<br />`key`: Your API key.<br />`weight`: Weight for upstream endpoint; positive number represents an endpoint weight, zero or negative disables this enpoint from routing. Default value: 1.<br />`tier`: Specifies tier group for the endpoint. Only positive numbers allowed. All requests will be routed to the endpoints with hieghest tier, other endpoints (with lower tier) may be used if and only if hiehest tier endpoints are unavailable. Default value: 1                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                        |
```

Comment:
Any chance to make it backward compatible to work with the previous config?

Reply:
if no weight and tier provided it will work as previous one